models/library-imported-from-ecore.json

```diff
@@ -19,234 +19,234 @@
     }
   },
   {
-    "type": "-EGokr9qbz6C-K8qdK_PHA",
+    "type": "LIonCore_M3_PrimitiveType",
     "id": "INhBvWyXvxwNsePuX0rdNGB_J9hi85cTb1Q0APXCyJ0",
     "properties": {
-      "EvXCQydomgm9DUb1IwkgRg": "String"
+      "LIonCore_M3_NamespacedEntity_simpleName": "String"
     }
   },
   {
-    "type": "-EGokr9qbz6C-K8qdK_PHA",
+    "type": "LIonCore_M3_PrimitiveType",
     "id": "uHmTCy63BV23cvXwFSumVwK1DOh4IZvIyAnRJjW9eQI",
     "properties": {
-      "EvXCQydomgm9DUb1IwkgRg": "boolean"
+      "LIonCore_M3_NamespacedEntity_simpleName": "boolean"
     }
   },
   {
-    "type": "-EGokr9qbz6C-K8qdK_PHA",
+    "type": "LIonCore_M3_PrimitiveType",
     "id": "gVp8_QSmXE2k4pd-sQZgjYMoW95SLLaVIH4yMYqqbt4",
     "properties": {
-      "EvXCQydomgm9DUb1IwkgRg": "int"
+      "LIonCore_M3_NamespacedEntity_simpleName": "int"
     }
   },
   {
-    "type": "jNyncSWEXyHd08N63CNBBA",
+    "type": "LIonCore_M3_Concept",
     "id": "OcDK2GESljInG-ApIqtkXUoA2UeviB97u0UuiZzM0Hs",
     "properties": {
-      "EvXCQydomgm9DUb1IwkgRg": "Book",
-      "j1tZfW_pl9Cu2NO3TUw4oA": false
+      "LIonCore_M3_NamespacedEntity_simpleName": "Book",
+      "LIonCore_M3_Concept_abstract": false
     },
     "children": {
-      "fSSOzJzcuK14nt-E3K2WTg": [
+      "LIonCore_M3_FeaturesContainer_features": [
         "Ei9m_HbdmEYg_EwMLLVZ71ERRBZyXH8GHVyOVia8Sqg",
         "OJhF8vB_qRMrA8A9a-H0LsByONdYtHGmQ-lk9rUHUc4",
         "9ATCb8nXEuQAL3NOY-EUZwuThpIQnQ2C_2n97IRGA_g"
       ]
     },
     "references": {
-      "noALNU7IxFfZDvrEL840Ig": [],
-      "wUjeOwaFMJPJX5IzUj6WWA": []
+      "LIonCore_M3_Concept_extends": [],
+      "LIonCore_M3_Concept_implements": []
     }
   },
   {
-    "type": "8zwB8soE_0a7rDVPgyxaRQ",
+    "type": "LIonCore_M3_Property",
     "id": "Ei9m_HbdmEYg_EwMLLVZ71ERRBZyXH8GHVyOVia8Sqg",
     "properties": {
-      "EvXCQydomgm9DUb1IwkgRg": "title",
-      "dAp79oxccBi-9j6BrQcZLQ": false,
-      "EE8DYkLRyQzaqyxFqlbOAw": false,
-      "XzYEKJ6QzlZuqKPtMXK_kQ": false
+      "LIonCore_M3_NamespacedEntity_simpleName": "title",
+      "LIonCore_M3_Feature_optional": false,
+      "LIonCore_M3_Feature_derived": false,
+      "LIonCore_M3_Property_disputed": false
```

Comment:
what is `Property.disputed`?

Reply:
It indicates for a property whether that property is “disputed” in the sense that
it's not in the agreed-on version of M3/LIonCore, but it's necessary
in order to be able to self-define LIonCore.